### PR TITLE
Websocket-Server and Webserver

### PR DESCRIPTION
**Pull request by dominikmatt**

### CODE REVIEW

File: server/config/core.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+const path = require('path');
+const glob = require("glob");
```
Comment (mob8607) [inline server/config/core.js:5]: use single quotes

Comment (mob8607) [reply server/config/core.js:5]: check for whole file

File: server/config/core.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    }
+
+    /**
+     * Returns Service by given Name.
+     *
+     * @param {string} name
+     * @returns {*}
+     */
+    getService(name) {
+        console.log(name);
```
Comment (mob8607) [inline server/config/core.js]: i think we should already start to implement a debug mode or something like that

File: web/js/Services/ConnectionService.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ * @type {Symbol}
+ */
+let singletonEnforcer = Symbol();
+
+/**
+ * @class ConnectionService
+ */
+class ConnectionService {
+    constructor(enforcer) {
+        if (enforcer != singletonEnforcer) throw 'Cannot construct singleton ConnectionService';
```
Comment (mob8607) [inline web/js/Services/ConnectionService.js:20]: use typesafe if

**Review by mob8607 — APPROVED** [review]
else ack

**Comment by dominikmatt** [comment]
done

**Comment by mob8607** [comment]
ack

**Pull request merged**